test/cluster/test_view_building_coordinator.py

```diff
@@ -92,3 +92,84 @@ async def test_view_building_with_tablet_split(manager: ManagerClient):
         await wait_for_view_is_built(cql, ks, 'mv')
         rows = await cql.run_async(f"SELECT * FROM {ks}.mv")
         assert len(list(rows)) == keys
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_increase_rf_during_view_building(manager: ManagerClient):
```

Comment:
These tests won't work with rf-valid keyspaces yet, and they can't since we can't handle changing rf or adding or removing racks, but maybe, to avoid more changes later, we can use racks already in this test